### ISSUE TITLE
Object modifiers in 2.5.59.13 not as expected

### ISSUE THREAD

**Issue by CuredPrusa** [supermerill/Slic3r#4437]
I doubt this is version related.
The object can be modified with infill (for example) just fine, as expected:
![image](https://github.com/user-attachments/assets/be02dc45-fc6f-4e10-a8d8-4a63f2d91ec4)
![image](https://github.com/user-attachments/assets/30c501e6-bce8-4fe0-92fc-85b0c0a64cf4)

But when I go with fuzzy skin, it starts introducing perimeters of the modifier:
![image](https://github.com/user-attachments/assets/2a5f5331-0911-4a31-9ccc-5e616ef1af05)
![image](https://github.com/user-attachments/assets/bdcccce9-5bff-431f-842c-94730984f946)
![image](https://github.com/user-attachments/assets/cee17054-4980-481e-932b-9b4ee2bde835)

I would like to have just one side of the cylinder fuzzy. No other unwanted artefacts. I see that Orca is handling the same situation much better, but I can't use Orca because the Orca UI is driving me crazy with all those unnecessary options in settings, variations and subdivisions of subdivisions in profiles.  

Why is this happening with fuzzy skin in SS? Is there a workaround? Did I do something wrong?
Are there any other modifying options which would induce the same problem (so I can prepare myself)?

**Comment by neophyl** [comment]
That’s exactly as expected with both PS and SuSi. Anything that involves different perimeters does that, as using a modifier splits it into separate slicing volumes so you get the perimeters from both. 

There are many issues on the PRUSA slicer GitHub about it and I’m sure if you searched here you will find more.

**Comment by CuredPrusa** [comment]
Thanks for the answer. So I guess I should live with it.
How come Orca made it work so smoothly?
Orca slices my project in 7 seconds, SS in 15s (after I have deleted appdata folder when I was waiting for more than 40sec for 1,5hour print job to slice). Orca delivers 11MB file, SS 22MB. I guess that could be due to 76.912 instances of M106 commands in SS gcode, vs 294 instances of M106 commands in Orca.

I wish SuperSlicer maintains its strength in this powerful development of other slicers.  I can navigate and tweak SS settings blindfolded, I'd really like to stick to it. Orca's UI looks like deliberately messed up, for programmers to show off how smart they are. Software should be made for user, not just for a sole purpose of creation of it. I wonder if it's obvious I got recently frustrated by Orca. :)
